### PR TITLE
Separate "Error ..." stacks for sample errors and errnos

### PR DESCRIPTION
**Pull request by Jongy**

### CODE REVIEW

File: examples/cpp/pyperf/PyPerfCollapsedPrinter.cc [hunk 1]
```diff
@@ -85,7 +85,7 @@ void PyPerfCollapsedPrinter::processSamples(
       truncatedStack++;
       break;
     case STACK_STATUS_ERROR:
-      std::fprintf(output_file, ";[Error %d]_[pe]", sample.errorCode);
+      std::fprintf(output_file, ";[Error (sample error) %d]_[pe]", sample.errorCode);
```
Comment (d3dave) [inline examples/cpp/pyperf/PyPerfCollapsedPrinter.cc:88]:
```suggestion
      std::fprintf(output_file, ";[Sample Error %d]_[pe]", sample.errorCode);
```

File: examples/cpp/pyperf/PyPerfCollapsedPrinter.cc [hunk 2]
```diff
@@ -97,7 +97,7 @@ void PyPerfCollapsedPrinter::processSamples(
           std::fprintf(output_file, ";(missing)_[pe]");
         }
         else {
-          std::fprintf(output_file, ";[Error %d]_[pe]", -stackId);
+          std::fprintf(output_file, ";[Error (errno) %d]_[pe]", -stackId);
```
Comment (d3dave) [inline examples/cpp/pyperf/PyPerfCollapsedPrinter.cc:100]: This can just stay "Error %d"

**Review by d3dave — CHANGES_REQUESTED** [review]

File: examples/cpp/pyperf/PyPerfCollapsedPrinter.cc (hunk 2 above)
Comment (Jongy) [reply examples/cpp/pyperf/PyPerfCollapsedPrinter.cc:100]: I like the mentioning of errno, helps reminding what numbers am I looking at

Comment (d3dave) [reply examples/cpp/pyperf/PyPerfCollapsedPrinter.cc:100]: But it's not errno, it's or'ed errnos. So the number as is doesn't necessarily reflect a single errno.

Comment (Jongy) [reply examples/cpp/pyperf/PyPerfCollapsedPrinter.cc:100]: `Error (errnos)`? :)

**Review by d3dave — APPROVED** [review]

**Pull request merged**